server/config/main.go

```diff
@@ -43,3 +47,29 @@ func (c *Configuration) IsValid() error {
 
 	return nil
 }
+
+func (c *Configuration) Sanitize() {
+	// Ensure Confluence ends with a slash
+	c.ConfluenceURL = strings.TrimRight(c.ConfluenceURL, "/")
```

Comment:
Comment indicates the opposite of what the code is doing. Should the URL end with a slash or not?